Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -239,6 +239,10 @@ extension NSManagedObjectContext: MessageDatabaseSession {
             parentMessageDTO.replies.insert(message)
         }
         
+        channelDTO.lastMessageAt = createdDate
+        channelDTO.updatedAt = createdDate
```

Comment:
The `updatedAt` shouldn't be changed. I think this value doesn't change with new messages, it's related to members and channel details.